README.md

```diff
@@ -1,3 +1,5 @@
+<img alt="redvin" src="./assets/readme/ipnn.svg" width="150" />
```

Comment:
move this into a seperate commit, for history :)

Reply:
okayy